RedBookPlayer.Models/Hardware/Player.cs

```diff
@@ -1332,37 +1338,79 @@ private void SoundOutputStateChanged(object sender, PropertyChangedEventArgs e)
     
         #region Helpers
 
+        /// <summary>
+        /// Parse multiple subchannels into object data
+        /// </summary>
+        /// <param name="subchannelData">Raw subchannel data to format</param>
+        /// <returns>List of subchannel object data</returns>
+        private List<SubchannelData> ParseSubchannels(byte[] subchannelData)
+        {
+            if(subchannelData == null || subchannelData.Length % 96 != 0)
+                return null;
+
+            // Create the list of objects to return
+            var parsedSubchannelData = new List<SubchannelData>();
+
+            // Read in 96-byte chunks
+            int modValue = subchannelData.Length / 96;
+            for(int i = 0; i < modValue; i++)
+            {
+                byte[] buffer = new byte[96];
+                Array.Copy(subchannelData, i * 96, buffer, 0, 96);
+                var singleSubchannel = new SubchannelData(buffer);
+                parsedSubchannelData.Add(singleSubchannel);
+            }
+
+            return parsedSubchannelData;
+        }
+
         /// <summary>
         /// Reformat raw subchannel data for multiple sectors
         /// </summary>
         /// <param name="subchannelData">Raw subchannel data to format</param>
         /// <returns>Dictionary mapping subchannel to formatted data</returns>
-        public Dictionary<char, byte[]> ConvertSubchannels(byte[] subchannelData)
+        private Dictionary<char, byte[]> ConvertSubchannels(byte[] subchannelData)
```

Comment:
private modifier is implicit

Reply:
I can provide additional cleanup of this in a later PR. The use of `private` is across a lot of places.